packages/aws-cdk-lib/aws-codepipeline/README.md

```diff
@@ -419,6 +421,52 @@ for details on how to use the variables for each action class.
 See the [CodePipeline documentation](https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-variables.html)
 for more details on how to use the variables feature.
```

Comment:
Lets add an example for an action-level variable here too to have parity with the pipeline-level section

Reply:
I added. 

https://github.com/aws/aws-cdk/pull/28538/commits/b2a03e7b69a7199c37e86df9e56a87204882cf5d